go/src/koding/kites/kloud/machine/client.go

```diff
@@ -0,0 +1,62 @@
+package machine
+
+import "time"
+
+// Machine represents a single machine.
+type Machine struct {
+	Team      string    `json:"team"`
+	Stack     string    `json:"stack"`
+	Provider  string    `json:"provider"`
+	Label     string    `json:"label"`
+	IP        string    `json:"ip"`
+	CreatedAt time.Time `json:"createdAt"`
+	Status    Status    `json:"status"`
+	Users     []User    `json:"users"`
+}
+
+// Status represents the current status of machine.
+type Status struct {
+	State      string    `json:"state"`
+	Reason     string    `json:"reason"`
+	ModifiedAt time.Time `json:"modifiedAt"`
+}
+
+// User represents a single user of described machine.
+type User struct {
```

Comment:
at least four things:
- security: if someone added sensitive data to models.MachineUser, we would leak these fields to kd client.
- symmetry: i think it's odd to have Machine struct here and User + Status from models.* package.
- consistency: credential package does the exact thing.
- idiomatic Go: [A little copying is better than a little dependency.](https://go-proverbs.github.io/)